setup.cfg

```diff
@@ -0,0 +1,45 @@
+#!/usr/bin/env python
+# coding=utf-8
+
+# THIS FILE IS PART OF THE CYLC SUITE ENGINE.
+# Copyright (C) 2008-2018 NIWA & British Crown (Met Office) & Contributors.
+#
+# This program is free software: you can redistribute it and/or modify
+# it under the terms of the GNU General Public License as published by
+# the Free Software Foundation, either version 3 of the License, or
+# (at your option) any later version.
+#
+# This program is distributed in the hope that it will be useful,
+# but WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+# GNU General Public License for more details.
+
+# You should have received a copy of the GNU General Public License
+# along with this program.  If not, see <http://www.gnu.org/licenses/>.
+
+[metadata]
+name = metomi-rose
+description = Rose, a framework for meteorological suites.
+author = 2012-2019 British Crown (Met Office) & Contributors
+author_email = metomi@metoffice.gov.uk
+url=https://metomi.github.io/rose/doc/html/index.html
+license = GPL
+license_file=COPYING
+keywords =
+    hpc
+    weather-modelling
+    weather-model
+    meteorological-suites
+    meteorological-modelling
+    meteorological-models
+platforms = any
+classifiers =
```

Comment:
I'm just cross-referencing with the setuptools config API for metadata, & I am not sure if this format is valid for ``classifiers`` (it is hard to tell from the documentation). Are you confident this works (can you test it)?

Reply:
The documentation is really a bit confusing. I wish someday someone would try to make it better organized (the content is good, just not so well organized I guess).

Doing a `pip install .`, then `pip show -f metomi-rose`, and finally confirming the `METADATA` file was generated, its contents are:

```
Metadata-Version: 2.1
Name: metomi-rose
Version: 2.0.0a3
Summary: Rose, a framework for meteorological suites.
Home-page: https://metomi.github.io/rose/doc/html/index.html
Author: 2012-2019 British Crown (Met Office) & Contributors
Author-email: metomi@metoffice.gov.uk
License: GPL
Keywords: hpc,weather-modelling,weather-model,meteorological-suites,meteorological-modelling,meteorological-models
Platform: any
Classifier: Environment :: Console
Classifier: Environment :: Web Environment
Classifier: Intended Audience :: Developers
Classifier: Intended Audience :: System Administrators
Classifier: Intended Audience :: Science/Research
Classifier: Operating System :: POSIX :: Linux
Classifier: Programming Language :: Python :: 3.7
Classifier: Topic :: Scientific/Engineering :: Atmospheric Science
Description-Content-Type: text/markdown
Requires-Dist: jinja2 (<2.11.0,>=2.10.1)
Requires-Dist: cylc-flow (==8.0a0)
Requires-Dist: aiofiles
Requires-Dist: tornado
Requires-Dist: sqlalchemy
Requires-Dist: isodatetime
Requires-Dist: requests
Requires-Dist: ldap3

# Rose

... (rest of readme)
```

The output appears to be OK, and the values used appear to be valid as well. But I guess I am not the best the review whether more or less values should be used in the classifiers... for reference, complete list (always have to search for "trove pypi classifiers" to find it) is here: https://pypi.org/pypi?%3Aaction=list_classifiers